eng/pipelines/common/global-build-job.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osSubgroup: ''
+  container: ''
+  testGroup: ''
+  crossrootfsDir: ''
+  timeoutInMinutes: ''
+  signBinaries: false
+  publishToBlobFeed: false
+  stagedBuild: false
+  variables: {}
+  pool: ''
+
+jobs:
+- template: /eng/common/templates/job/job.yml
+  parameters:
+    name: ${{ format('runtime_build_{0}{1}_{2}_{3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
+    displayName: ${{ format('Runtime Build {0}{1} {2} {3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
+    dependsOn: checkout
+    pool: ${{ parameters.pool }}
+    container: ${{ parameters.container }}
+
+    variables:
+    - name: configArg
+      value: "-c Release"
+    - ${{ if eq(parameters.buildConfig, 'debug') }}:
+      - name: configArg
+        value: "-c Debug"
+    - ${{ parameters.variables }}
+
+    steps:
+    - ${{ if eq(parameters.osGroup, 'OSX') }}:
+      - script: |
+          brew install pkgconfig icu4c openssl
```

Comment:
I'd add `set -ex` for stop on error and to show the commands in the logs as they're run.

Reply:
Fixed.